.redocly.yaml

```diff
@@ -15,25 +15,50 @@ apis:
       remove-x-internal: on
   admin-internal:
     root: "./docs/api/admin.yaml"
+  admin-capella:
+    root: "./docs/api/admin-capella.yaml"
+    decorators:
+      remove-x-internal: on
   public:
     root: "./docs/api/public.yaml"
     decorators:
       remove-x-internal: on
   public-internal:
     root: "./docs/api/public.yaml"
+  public-capella:
+    root: "./docs/api/public.yaml"
```

Comment:
should this be public-capella, or is the public-capella.yaml still in this PR now obsolete?

Reply:
I guess that question is "Did the filter-out x-capella thing work, or is that a failed experiment?"

Reply:
public-capella.yaml is dead, thanks for catch.

For filter-out x-capella, it does work, it's in `public.yaml` to remove the views APIs. Using `filter-out` the property has to be defined, so it can't just filter out everything that doesn't have the property defined. That's why admin and metric use different toplevel yaml files, since they have such limited APIs.

I've added filter-out x-capella preemptively for admin and metric endpoints, since we could filter out parts of a particular operation. We aren't doing that yet but could in the future.